velox/docs/configs.rst

```diff
@@ -524,6 +524,19 @@ Each query can override the config by setting corresponding query session proper
      - bool
      - false
      - Utilize the configuration of the environment variables http_proxy, https_proxy, and no_proxy for use with the S3 API.
+   * - hive.s3.connect-timeout
+     - string
+     - 5s
+     - TCP connect timeout.
+   * - hive.s3.socket-timeout
+     - string
+     - 5s
+     - TCP socket read timeout.
+   * - hive.s3.max-connection
+     - integer
+     - 500
```

Comment:
25

Reply:
I presume users will share catalog configs for Coordinator and Prestissimo. If we have different default value(not set this property), will it cause confusion?

Reply:
You mean the Java default config is 500? I did not understand why it would cause confusion since this property is not set.

Reply:
Let's stick with the SDK default until determine a good default value. We can specify here that we will use the CPP SDK default.

Reply:
Thanks @majetideepak , I updated the code to stick with SDK default value. Could you please take a look?